packages/lib/server/checkUsername.ts

```diff
@@ -1,6 +1,6 @@
-import { checkPremiumUsername } from "@calcom/features/ee/common/lib/checkPremiumUsername";
-import { IS_SELF_HOSTED } from "@calcom/lib/constants";
+import { IS_CALCOM } from "@calcom/lib/constants";
 
 import { checkRegularUsername } from "./checkRegularUsername";
+import { usernameCheck } from "./username";
 
-export const checkUsername = IS_SELF_HOSTED ? checkRegularUsername : checkPremiumUsername;
+export const checkUsername = !IS_CALCOM ? checkRegularUsername : usernameCheck;
```

Comment:
naming is tough here 😆 , how can we improve this? this is what went in my head when I read this: checkUsername is either checkRegularUsername or usernameCheck, what's the difference ?